### ISSUE TITLE
Make systems ID stable and unique

### ISSUE THREAD

**Issue by scheidan** [santiago-sanitation-systems/Santiago.jl#38]
Currently we simply number them, but tehy may change if we add techs

**Comment by spuhlerd** [comment]
to create a function that can select sys by ID... view_system or access_system(ID: vector)

**Comment by scheidan** [comment]
implemented by c0d6afedada505b85a40099cfff33a5affd97bcc

You can pick systems based on the new IDs:
```Julia
pick_systems(systems, ["003s-QbnU-FvGB", "0JLD-YQbJ-SGAu"])
```
